Tree.h

```diff
@@ -132,13 +132,21 @@ class LeafNode : public TreeNode<T> {
 
 // load a regression tree from Json
 template <class T>
-TreeNode<T>* fromJson(const folly::dynamic& obj) {
-  int index = obj["index"].asInt();
+TreeNode<T>* fromJson(const folly::dynamic& obj, const Config& cfg) {
+  const folly::dynamic* feature = nullptr;
+  try {
+    feature = &obj["feature"];
+  } catch(...) {
+  }
```

Comment:
I think there might be a bug in dynamic. The error message is that get is not implemented for const_iterator. The example works but in our code we pass in a const reference. Will submit it for now.